packages/apps/kadena-docs/src/components/Layout/Full/style.ts

```diff
@@ -0,0 +1,50 @@
+import { styled, StyledComponent } from '@kadena/react-components';
+
+export const AsideList: StyledComponent<
+  'ul',
+  { inner?: boolean | 'true' | 'false' | undefined }
+> = styled('ul', {
+  listStyle: 'none',
+  margin: 0,
+  padding: '0',
+
+  '& li::before': {
+    content: '∙',
+    color: '$primaryHighContrast',
+    fontWeight: '$bold',
+    display: 'inline-block',
+    width: '$sizes.$4',
```

Comment:
I don't think this works? Width will automatically use size tokens so you can just put `$4` but if you want to specify the group you don't need to add the period `$sizes$4`

Reply:
right. fixing it